src/content/docs/identityserver/upgrades/identityserver4-to-duende-identityserver-v7.mdx

```diff
@@ -0,0 +1,513 @@
+The exact steps involved in updating your data store will depend on your implementation details.
+
+- Create a new `Keys` table for the automatic key management feature in the operational database.
+- Create a new `RequireResourceIndicator` boolean column on the `ApiResources` table in the configuration database.
+- Create a new index on the `ConsumedTime` column in the `PersistedGrants` table ([more details](https://github.com/DuendeSoftware/products/pull/84)).
+- Create a new table called `IdentityProviders` for storing the OIDC provider details ([more details](https://github.com/DuendeSoftware/products/pull/188)).
+- Add missing columns for created, updated, etc. to EF entities ([more details](https://github.com/DuendeSoftware/products/pull/356)).
+- Add unique constraints to EF tables where duplicate records not allowed ([more details](https://github.com/DuendeSoftware/products/pull/355)).
+- The server-side sessions feature requires a new table ([more details](https://github.com/DuendeSoftware/products/pull/743)).
+- The session coordination feature adds a column to the `Clients` table ([more details](https://github.com/DuendeSoftware/products/pull/820)).
+- Improve primary keys on the persisted grants table ([more details](https://github.com/DuendeSoftware/products/pull/793)).
+- Add new properties to the [`Duende.IdentityServer.Models.Client` model](/identityserver/reference/models/client):
+
+  - `InitiateLoginUri` is a nullable string used for Third Party Initiated Login.
+  - `RequireDPoP` is a non-nullable boolean flag that controls if a client is required to use [DPoP](../../tokens/pop).
+  - `DPoPValidationMode` is a non-nullable column that controls the DPoP validation mechanism. Existing clients that are not using DPoP can set its value to `0`.
+  - `DPoPClockSkew` is a non-nullable timespan that controls how much clock skew is allowed for a particular DPoP client. Existing clients that are not using DPoP can set its value to a timespan of length ``0.
+
+- The `ServerSideSession` entity now uses a 64-bit long as its primary key.
+- Two new properties have been added to the `Client` model:
+  - `Client.RequirePushedAuthorization` is a new boolean property that controls if this client requires [pushed authorization requests (PAR)](../../tokens/par). It is safe to initialize this column to `false` for existing clients, which will mean that the global PAR configuration will be used.
+  - `Client.PushedAuthorizationLifetime` is a new nullable integer property that controls the lifetime of pushed
+    authorization requests (in seconds) for a client. It is safe to initialize this column to `null` for existing clients, which means the global value is used.
+- A new `PushedAuthorizationRequest` table has been added to store pushed authorization requests.
+
+In this section, we'll look at updating the database schema based on the stores provided in the `Duende.IdentityServer.EntityFramework` package.
```

Comment:
Consider moving this before the details listed. This is what most people will need - create and run a migration. All the details are really only relevant for someone having their own stores. And that's the exception case.